src/main/java/io/weidongxu/util/releaseautomation/LiteRelease.java

```diff
@@ -130,6 +130,9 @@ public static void main(String[] args) throws Exception {
 
             variables.put("README", new Variable().withValue(sdk));
             variables.put("TSP_CONFIG", new Variable().withValue(tspConfig.getUrl()));
+            if (!CoreUtils.isNullOrEmpty(configure.getVersion())) {
+                variables.put("VERSION", new Variable().withValue(configure.getVersion()));
+            }
```

Comment:
Do we do same as swagger?

https://github.com/weidongxu-microsoft/release-automation-devops/pull/14/files#diff-8d06e0dd2e42bb49a350d64c3fe3eb10500d151be97ff08deb6f2d65da78284aR201-R203

Reply:
OK, we don't able to figure out version as there is no tag and readme.

Reply:
We could infer sdk version from TypeSpec, if we pin api-version in tspconfig.yaml. 
Currently we don't pin api-version. Let's see if we need that in the future.